frappe/desk/search.py

```diff
@@ -114,42 +109,47 @@ def search_widget(
 					http_status_code=404,
 				)
 				return []
+	elif doctype in standard_queries:
+		query = standard_queries[doctype][-1]
```

Comment:
Doing this later will not run it at all... what am I missing?

Reply:
True. 😬

However, `test_link_search_in_foreign_language` doesn’t fail, so is not specific enough.

I could simply walk back this misguided change, and I will. But we better add another unit test for `doctype in standard_queries`.

Reply:
Walked it back.